### PR TITLE
Update Nook to new auth info

### PR DESCRIPTION
**Pull request by lukechurch**
This updates Nook to use https://github.com/larksystems/katikati_ui_lib.dart/pull/23

### CODE REVIEW

File: webapp/lib/view.dart [hunk 1]
```diff
@@ -72,8 +72,9 @@ class LoginPage {
   AuthMainView authView;
 
   LoginPage() {
-    authView = new AuthMainView(Brand.katikati, '', '', [SignInDomain.katikati, SignInDomain.lark],
-        (SignInDomain domain) => _pageView.appController.command(BaseAction.signInButtonClicked, new SignInData(domain)));
+    authView = new AuthMainView(Brand.katikati, '', '',
+    [KATIKATI_DOMAIN_INFO, LARK_DOMAIN_INFO],
+        (SignInDomainInfo domain) => _pageView.appController.command(BaseAction.signInButtonClicked, new SignInData(domain)));
```
Comment (marianamarasoiu) [inline webapp/lib/view.dart:77]: indentation nit:
```suggestion
        [KATIKATI_DOMAIN_INFO, LARK_DOMAIN_INFO],
        (SignInDomainInfo domain) => _pageView.appController.command(BaseAction.signInButtonClicked, new SignInData(domain)));
```

Comment (lukechurch) [reply webapp/lib/view.dart:77]: ah thanks

**Review by marianamarasoiu — APPROVED** [review]
LGTM with a nit, thanks @lukechurch !

**Comment by lukechurch** [comment]
thanks @marianamarasoiu !

**Pull request merged**